wordpress-mobile/WordPress-Android
# Remove Dangerfile, gem and CircleCI job

Danger has now been replaced with its self-hosted equivalent, [Peril](https://github.com/danger/peril).

All existing rules from the `Dangerfile` have been added to our new Peril settings repo: https://github.com/wordpress-mobile/peril-settings. This will allow us to share rules across projects and platforms.

To test:

- See that there is a new Peril status on this PR, and on the status page: https://github.com/wordpress-mobile/WordPress-Android/pull/9265/checks?check_run_id=66633543.

Update release notes:
